### PR TITLE
Add suport for multiple disks per kafka broker

### PR DESCRIPTION
**Pull request by nysthee**

### CODE REVIEW

**Comment by gAmUssA** [comment]
@nysthee Thank you for PR.
Kindly help me to understand what are you trying to achieve here? JBOD?
Cheers,
Vik

**Comment by nysthee** [comment]
Hi,
Indeed we were debugging an issue with a Confluent pre-sales engineer and he suggested to add multiple disks to our brokers. So I thought this would be useful for more people.

File: charts/cp-kafka/templates/statefulset.yaml [hunk 1]
```diff
@@ -153,18 +157,23 @@ spec:
       {{- end }}
   {{- if .Values.persistence.enabled }}
   volumeClaimTemplates:
+  {{- $disksPerBroker := .Values.persistence.disksPerBroker | int }}
+  {{- $root := . }}
```
Comment (qshao-pivotal) [inline charts/cp-kafka/templates/statefulset.yaml]: Do we need root here?

**Comment by qshao-pivotal** [comment]
This is definitely needed. Thanks a lot for the PR.

File: charts/cp-kafka/templates/statefulset.yaml (hunk 1 above)
Comment (nysthee) [reply charts/cp-kafka/templates/statefulset.yaml]: No indeed not necessary

**Comment by nysthee** [comment]
removed it!

**Comment by nysthee** [comment]
@qshao-pivotal  you can verify if you want

**Comment by qshao-pivotal** [comment]
I will. Thanks

**Comment by qshao-pivotal** [comment]
Also Need to surface `disksPerBroker` to cp-helm-charts' `values.yaml`

**Comment by nysthee** [comment]
ok done

File: charts/cp-kafka/templates/statefulset.yaml [hunk 2]
```diff
@@ -101,6 +101,10 @@ spec:
           value: {{ .Values.heapOptions }}
         - name: KAFKA_ZOOKEEPER_CONNECT
           value: {{ include "cp-kafka.cp-zookeeper.service-name" . | quote }}
+        {{- if not (hasKey .Values.configurationOverrides "log.dirs") }}
```
Comment (AzureQ) [inline charts/cp-kafka/templates/statefulset.yaml]: This if condition seems no longer needed.

**Comment by gAmUssA** [comment]
@nysthee while I'm working on automating PR checks, could you run linting and testing scripts in scripts directory?

**Comment by nysthee** [comment]
image

Do you want me to package as well?

**Comment by gAmUssA** [comment]
LGTM! Sure, send (separate, if it's not too much of trouble) PR with binary. 
Thank you

**Comment by nysthee** [comment]
Will you merge this first?

**Pull request merged**